### DISCUSSION TITLE
Search Operators

### DISCUSSION THREAD

**Discussion by briandeadalive** [tiekoetter/searxng#27] [Q&A]
Thank you for all of the work you do and for maintaining this resource.

I’m trying to figure out what search operators are most effective with searx.

I’m currently using boolean operators like “xxxxx” for exact keywords, AND, OR, etc. However, I’m unsure if they are actually doing anything.

The only relevant info I could find states that there isn’t a standardized method for creating a more specific search because searx is a meta search engine so it depends on which engines I enabled. 

I read the info on ‘bangs!’ included in the ‘user instruction’ module in this instances’ github wiki, but it doesn’t provide the clarity I’m seeking. 

Do you have any suggestions for which search operators may be most effective across the wide spectrum of search engines?

Thank you for your service.

(Side note:  Speaking as a noob, it could be helpful to include a hyperlink to the ‘user instruction’ module along with the existing links in the footer below the instance’s search bar.)

**Comment by tiekoetter** [comment]
You are correct there is no standardized way to really make a specific "filtered" search. You mostly depend on what the upstream engine supports. Any "across the board" implementation would be vary hard I think.
